utils/find-wasm.js

```diff
@@ -0,0 +1,42 @@
+const util = require('util');
+const glob = util.promisify(require('glob'));
+const fs = require('fs');
+
+
+
+async function findWasm(target) {
+    let res = await glob(`**/${target}/*.wasm`, { 
+        ignore: '**/node_modules/**'
+    });
+    return res;
+}
+
+async function findWasmFile(wasmFile, target) {
+    try {
+        return findWasmFileSafe(wasmFile, target);
+    } catch (err) {
+        console.error(err.message);
+        process.exit(1);
+    }
+}
+
+async function findWasmFileSafe(wasmFile, target) {
+    if (fs.existsSync(wasmFile)) {
+        return wasmFile;
+    }
+    if (wasmFile === './out/main.wasm') {
```

Comment:
I still think we should probably remove the default from the `deploy` and `dev-deploy` commands, and add some sort of concise description to the help output for these commands, explaining this behavior. But maybe it's worth getting a second opinion.

@mikedotexe what do you think? This change is going to make it possible to remove the symlinking-to-`out/main.wasm` behavior from `create-near-app`'s `compile` scripts, and just find the single `*.wasm` file in the `contract` folder. It also sets us up to add interactivity to near-cli later, so if you have a project with multiple contracts, it can list your Wasm files for you and ask which you'd like to deploy. 

Right now, the help docs for `near deploy --help` and `near dev-deploy --help` say:

```
--wasmFile, --wasm_file         Path to wasm file to deploy  [string] [default: "./out/main.wasm"]
```

Should we remove the default now? Should we make it say something like:

```
--wasmFile, --wasm_file         Path to wasm file to deploy. Unless specified, looks for "./out/main.wasm" or "**/release/*.wasm"  [string]
```

Reply:
I feel like leaving the default makes sense as we have a number of examples that'll work seamlessly. Feels like what you're suggesting might add one (small) extra step

Reply:
My suggestion is not to change the default behavior, only to remove the explicit default in the code and to document the truer behavior of this improved interface.

Reply:
I think Chad makes a good point that we can remove the `default` key and change the help text. I didn't realize this would be backwards-compatible. Good find!

Reply:
Note that we would want to remove the `default` and update the `desc` for both `deploy` and `dev-deploy`. Here's the `desc` I recommend for each:

* `deploy`: File to deploy. Default: "./out/main.wasm" or "**/release/*.wasm"
* `dev-deploy`: File to deploy. Default: "./out/main.wasm" or "**/debug/*.wasm"